dhis-2/dhis-api/src/main/java/org/hisp/dhis/trackedentity/TrackedEntityProgramOwnerService.java

```diff
@@ -40,79 +40,79 @@ public interface TrackedEntityProgramOwnerService {
    * Assign an orgUnit as the owner for a tracked entity instance for the given program. If another
    * owner already exist then this method would fail.
    *
-   * @param teiUid The Uid of the tracked entity instance
+   * @param teUid The Uid of the tracked entity instance
```

Comment:
```suggestion
   * @param teUid The Uid of the tracked entity
```

Reply:
There are also some lost `instance` in the description of the JavaDocs

Reply:
yes good spot